integrations/gitlab/gitlab_integration/ocean.py

```diff
@@ -102,10 +102,7 @@ async def on_start() -> None:
         await event_handler.start_event_processor()
         await system_event_handler.start_event_processor()
     except Exception as e:
-        logger.warning(
-            f"Failed to setup webhook: {e}. {NO_WEBHOOK_WARNING}",
-            stack_info=True,
-        )
+        logger.exception(f"Failed to setup webhook: {e}. {NO_WEBHOOK_WARNING}")
```

Comment:
need to make sure that it actually prints the stacktrace, I think `exception` does it but just lets make sure